pkg/controller/user-cluster-controller-manager/rbac/resources_test.go

```diff
@@ -32,11 +32,13 @@ func TestGeneratedResourcesForGroups(t *testing.T) {
 		resourceName      string
 		expectError       bool
 		expectedResources []string
+		expectedVerbs     []string
```

Comment:
the `expectedVerbs` are not used anywhere from what i see?

Reply:
oh yeah I missed cleaning that up. Good spot!